core/domain/contribution_stats_services.py

```diff
@@ -252,16 +253,19 @@ def get_translation_submitter_total_stats(
             result.
         topic_ids: List[str]|None. List of topic ID(s) to fetch
             contributor stats for.
-        max_days_since_last_activity: int. To get number of users
-            who are active in max_days_since_last_activity.
+        max_days_since_first_activity: int. Number of days before today
+            to start including users with contributions.
+        max_days_since_last_activity: int. Number of days before today
+            from which to stop including users with contributions.
```

Comment:
Instead of using the terms `first_activity` and `last_activity`, can we instead use `start_date_for_last_activity` and `end_date_for_last_activity` throughout the relevant backend code?

Reply:
done